cpp/tests/reshape/interleave_columns_tests.cpp

```diff
@@ -374,4 +368,298 @@ TYPED_TEST(FixedPointTestBothReps, FixedPointInterleave)
   }
 }
 
+namespace {
+using StrListsCol = cudf::test::lists_column_wrapper<cudf::string_view>;
+using IntListsCol = cudf::test::lists_column_wrapper<int32_t>;
+using IntCol      = cudf::test::fixed_width_column_wrapper<int32_t>;
+using TView       = cudf::table_view;
+
+constexpr bool print_all{true};  // For debugging
+constexpr int32_t null{0};
+
+auto null_at(cudf::size_type idx)
```

Comment:
Please use `iterator_with_null_at()` from `cudf_test/iterator_utilities.hpp`.

Reply:
Unfortunately `iterator_with_null_at` is so long. That's why I didn't use it :(

Reply:
Another problem with that header is that, if included then all the functions in it must be used, otherwise I will got compilation error "defined but not used".

Reply:
I'm filling another PR for this.

Reply:
> Unfortunately `iterator_with_null_at` is so long.

Pardon me, but the fact that one does not like the name of the function cannot be the reason to copy and repeat code. This is contradictory to your own general philosophy of preferring brevity and avoiding repetition. :]
(FWIW, the code started out being called `null_at()`. It was changed to `iterator_with_null_at()`, based on advice during the review.)

> I can rewrite them into inline functions to avoid that error...

This is an interesting point. If all it takes is to inline the function, by all means do. I'm supportive of a tiny fix here, if it promotes (re)usability.